garage/tf/policies/categorical_mlp_policy.py

```diff
@@ -21,14 +21,20 @@ def __init__(
             prob_network=None,
     ):
         """
-        :param env_spec: A spec for the mdp.
-        :param hidden_sizes: list of sizes for the fully connected
-        hidden layers
-        :param hidden_nonlinearity: nonlinearity used for each hidden layer
-        :param prob_network: manually specified network for this policy, other
-         network params
-        are ignored
-        :return:
+        CategoricalMLPPolicy.
+
+        A policy that uses a MLP to estimate a categorical distribution.
+
+        Args:
+            env_spec (garage.envs.env_spec.EnvSpec): Environment specification.
+            hidden_sizes (list[int]): Output dimension of dense layer(s).
+                For example, (32, 32) means the MLP of this policy consists
+                of two hidden layers, each with 32 hidden units.
+            hidden_nonlinearity: Activation function for
```

Comment:
there are `tf.Operation` right?

Reply:
I don't think they are tf.Operation. In python doing type(tf.nn.relu) returns <class 'function'>.

Reply:
it's a function which returns tf.Tensor https://github.com/tensorflow/tensorflow/blob/r1.13/tensorflow/python/ops/nn_ops.py#L2011

See https://mypy.readthedocs.io/en/latest/cheat_sheet_py3.html#functions for a guide on how to write the type hint for a function